### DISCUSSION TITLE
Sensore stato contatto/circuito

### DISCUSSION THREAD

**Discussion by Matty1184** [indomus/forum#2060] [Hardware & Firmware]
Ciao, vorrei installare un dispositivo che mi rilevi se un contatto è aperto o chiuso e che non faccia da relè, mi spiego meglio...
l'antifurto in mio possesso non è integrabile, ma ho la possibilità di avere tre uscite (tre contatti puliti) per far si che mi possa dare:
stato settore1 inserito/disinserito (chiude contatto se inserito)
stato settore2 inserito/disinserito (chiude contatto se inserito)
stato in allarme (chiude contatto se allarme si attiva)
vorrei leggerli e importarli in HA.
che dispositivi mi consigliate?

**Comment by dante70** [comment]
Ciao,io vedo al volo 2 possibilita semplici:
monti un D1 mini e mandi a 3 dei suoi GPIO i contatti dei rele' che hai.
programmato con esphome e' molto semplice e ha costi  di pochi euro
diversamente e altrettanto semplice ,anzi dipiu',monti un shelly i3 che ti espone lo stato dei 3 ingressi su HA
